### DISCUSSION TITLE
Could ASUS Services affect asus-fan-control?

### DISCUSSION THREAD

**Discussion by sw5ciprl** [dominiksalvet/asus-fan-control#100] [Q&A]
After reading some of the info, I found out that ASUS' own services restrict editing of certain values if enabled. Could it be possible that these services are set up so we can't change fan speed while they're installed?

**Comment by dominiksalvet** [comment]
I haven't heard about that but I can imagine it is possible. I doubt, however, that the services are from ASUS itself -- they only have the brand in their name. Nevertheless, I am not an expert on these things.
